build.sbt

```diff
@@ -53,11 +53,12 @@ ThisBuild / crossScalaVersions := Seq(Scala212, Scala3, Scala213)
 val catsEffectVersion = "3.5.4"
 val circeVersion = "0.14.7"
 val fs2Version = "3.10.2"
-val http4sVersion = "0.23.27"
+val http4sVersion = "0.23.27-10-fa6e976-SNAPSHOT"
```

Comment:
We'll need to wait for http4s v0.23.28 before we can release, but we're good to merge and publish a snapshot!